File: app/src/main/java/com/kproduce/roundcorners/test/MainActivity.java
package com.kproduce.roundcorners.test;


import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.view.View;

import com.kproduce.roundcorners.RoundTextView;

/**
 * @author kuanggang on 2019/12/01
 */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        final RoundTextView tv = findViewById(R.id.tv);
        tv.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                tv.setRadius(10, 0, 0, 0);
                tv.setStrokeWidthColor(5, getResources().getColor(android.R.color.holo_green_dark));
            }
        });
    }
}

File: app/src/main/java/com/kproduce/roundcorners/test/MainActivity.kt
package com.kproduce.roundcorners.test

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

/**
 * @author kuanggang on 2019/12/01
 */
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        tv.setOnClickListener {
            tv.setRadius(10f, 0f, 0f, 0f)
            tv.setStrokeWidthColor(5f, resources.getColor(android.R.color.holo_green_dark))
        }
    }
}